lib/extensions/require_nested.rb

```diff
@@ -15,6 +15,14 @@ def require_nested(name)
       Object.require_nested name
     end
   end
+
+  def require_nested_all
+    path = caller(1, 1).first.split('.rb').first + '/*.rb' # determining caller file location
+    Dir.glob(path).sort.each do |full_path|
+      name = File.basename(full_path, '.rb')
+      require_nested(name.camelize.to_sym)
```

Comment:
Minor, but prefer `.classify` over `.camelize` here.  Almost exactly the same, but handles namespaces and such, though it will likely make no difference.

Reply:
done!